### DISCUSSION TITLE
Can I use Forecaster as a baseline predictor?

### DISCUSSION THREAD

**Discussion by XaviMartiCamarasa** [online-ml/river#1093] [Q&A] [answered]
Hi.
Looking for a baseline predictor. I'm guessing if I could use Forecaster as a baseline or not.

Thanks in advance

**Comment by MaxHalford** [comment, marked as answer]
Hey there.

Do you mean you're looking for a baseline model for time-series forecasting? `Forecaster` is a base class, it's not meant to be used as is, but rather other models inherit from it.

**Comment by XaviMartiCamarasa** [comment]
Many thanks
Any idea to use as a baseline forecaster ?


El mar., 22 nov. 2022 16:43, Max Halford ***@***.***>
escribió:

> Hey there.
>
> Do you mean you're looking for a baseline model for time-series
> forecasting? Forecaster is a base class, it's not meant to be used as is,
> but rather other models inherit from it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/online-ml/river/discussions/1093#discussioncomment-4207691>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A37HEL6YF4Z22RJFI4NEOJLWJTS2VANCNFSM6AAAAAASH5NMFU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 



INFORMACIÓ SOBRE PROTECCIÓ DE DADES DE LA UNIVERSITAT OBERTA DE 
CATALUNYA (UOC)

Us informem que les vostres dades identificatives i les 
contingudes en els missatges electrònics i fitxers adjunts es poden 
incorporar a les nostres bases de dades amb la finalitat de gestionar les 
relacions i comunicacions vinculades a la UOC, i que es poden conservar 
mentre es mantingui la relació. Si ho voleu, podeu exercir el dret a 
accedir a les vostres dades, rectificar-les i suprimir-les i altres drets 
reconeguts normativament adreçant-vos a l'adreça de correu emissora o a 
***@***.*** ***@***.***>.

Aquest missatge i qualsevol 
fitxer que porti adjunt, si escau, tenen el caràcter de confidencials i 
s'adrecen únicament a la persona o entitat a qui s'han enviat.

Així 
mateix, posem a la vostra disposició un delegat de protecció de dades que 
no només s'encarregarà de supervisar tots els tractaments de dades de la 
nostra entitat, sinó que us podrà atendre per a qualsevol qüestió 
relacionada amb el tractament de dades. La seva adreça de contacte és 
***@***.*** ***@***.***>.
INFORMACIÓN SOBRE PROTECCIÓN DE DATOS DE 
LA UNIVERSITAT OBERTA DE CATALUNYA (UOC)
Os informamos de que vuestros 
datos identificativos y los contenidos en los mensajes electrónicos y 
ficheros adjuntos pueden incorporarse a nuestras bases de datos con el fin 
de gestionar las relaciones y comunicaciones vinculadas a la UOC, y de que 
pueden conservarse mientras se mantenga la relación. Si lo deseáis, podéis 
ejercer el derecho a acceder a vuestros datos, rectificarlos y suprimirlos 
y otros derechos reconocidos normativamente dirigiéndoos a la dirección de 
correo emisora o a ***@***.*** ***@***.***>.
Este mensaje y 
cualquier fichero que lleve adjunto, si procede, tienen el carácter de 
confidenciales y se dirigen únicamente a la persona o entidad a quien se 
han enviado.
Así mismo, ponemos a vuestra disposición a un delegado de 
protección de datos que no solo se encargará de supervisar todos los 
tratamientos de datos de nuestra entidad, sino que podrá atenderos para 
cualquier cuestión relacionada con el tratamiento de datos. Su dirección de 
contacto es ***@***.*** ***@***.***>.


UNIVERSITAT OBERTA DE 
CATALUNYA (UOC) DATA PROTECTION INFORMATION
Your personal data and the data 
contained in your email messages and attached files may be stored in our 
databases for the purpose of maintaining relations and communications 
linked to the UOC, and the data may be stored for as long as these 
relations and communications are maintained. If you so wish, you can 
exercise your rights to access, rectification and erasure of your data, and 
any other legally held rights, by writing to the sender’s email address or 
***@***.*** ***@***.***>.
This message and, where 
applicable, any attachments are confidential and addressed solely to the 
individual or organization they were sent to.
The UOC has a data protection 
officer who not only supervises the data processing carried out at the 
University, but who will also respond to any questions you may have about 
this data processing. You can contact our data protection officer by 
writing to ***@***.*** ***@***.***>.

**Reply by MaxHalford** [reply]
[HoltWinter](https://riverml.xyz/0.14.0/api/time-series/HoltWinters/) is a good baseline for forecasting.

**Reply by XaviMartiCamarasa** [reply]
Many many thanks Max!

**Reply by MaxHalford** [reply]
No problem =)
